appveyor.yml

```diff
@@ -5,7 +5,7 @@ environment:
     GRADLE_OPTS: -Dorg.gradle.daemon=false
 
     # URL for SDK Tools, Revision 26.1.1 (September 2017)
```

Comment:
I guess the version here is no longer correct with this change

Reply:
Supposedly 26.1.1 is still the latest, but the download link doesn't match what was here before :confused: